### PR TITLE
Uniquenessness

### PR DESCRIPTION
**Pull request by roberttod**
Generates a random ID in the form `iiiiiiiiiii-ttttttttt-fffffff` using a random integer, a timestamp and a fingerprint. The fingerprint is an attempt to uniquely identify the user's browser using `navigator.plugins`, `navigator.userAgent` and `document.cookie`. The timestamp and random integer are encoded at base36 to keep the ID as small as possible.

### CODE REVIEW

**Comment by roberttod** [comment]
@KidkArolis @jhollingworth

**Comment by jhollingworth** [comment]
hate to be pedantic but this isn't very random. [Randomness is a lot more clumpy](http://www.wired.com/2012/12/what-does-randomness-look-like/). Could we rename this this repo "slightly-random-id"?

File: really-random-id.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+/**
+ * Returns the time in ms since 1970.
+ *
+ * @returns {Number} hammertime
+ */
+function hammerTime () {
+  /**
+   * http://en.wikipedia.org/wiki/List_of_dates_predicted_for_apocalyptic_events
+   * According to an opinion on the Talmud in mainstream
```
Comment (jhollingworth) [inline really-random-id.js]: best to leave jokes for internal repos.

Comment (roberttod) [inline really-random-id.js]: :older_man:

**Comment by jhollingworth** [comment]
uniqueness is a binary quality. You don't have different grades of uniqueness. It is either unique or not

**Comment by roberttod** [comment]
Almost unique ID? It was kind of meant to be playful...

**Pull request merged**